app/components/admin/sdg/targets/index_component.html.erb

```diff
@@ -0,0 +1,38 @@
+<% provide(:title) do %>
+  <%= t("admin.header.title") %> - <%= title %>
+<% end %>
+
+<h2><%= title %></h2>
+
+<%= link_to t("admin.sdg.goals.index.title"), admin_sdg_goals_path %>
+<%= link_to t("admin.sdg.targets.index.title"), admin_sdg_targets_path %>
+
+<table>
+  <thead>
+    <tr>
+      <th></th>
```

Comment:
I think the [WAI example](https://www.w3.org/WAI/tutorials/tables/examples/threeheaders/) used an empty `<td>` here, right?